### PR TITLE
[haywardomnilogic] Update unit formating

### PR DESCRIPTION
**Pull request by matchews**
Signed-off-by: Matt Myers <mmyers75@icloud.com>

Minor formatting updates and new code check fixes

### CODE REVIEW

File: bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/backyard.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
 		<item-type>Number:Temperature</item-type>
 		<label>Air Temp</label>
 		<description>Air Temp</description>
-		<state pattern="%1f %unit%" readOnly="true"/>
+		<state pattern="%1.0f%unit%" readOnly="true"/>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/backyard.xml:36]: Why did you remove the space between the value and the unit? Please add it again, we have that for all bindings. Same of the other occurrences.

File: bundles/org.openhab.binding.haywardomnilogic/src/main/feature/feature.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <features name="org.openhab.binding.haywardomnilogic-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
 	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
 
-	<feature name="openhab-binding-haywardomnilogic" description="Hayward OmniLogic Binding" version="${project.version}">
+	<feature name="openhab-binding-haywardomnilogic" description="haywardomnilogic Binding" version="${project.version}">
```
Comment (fwolter) [inline bundles/org.openhab.binding.haywardomnilogic/src/main/feature/feature.xml:5]: Is this change intentional?

File: bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/colorlogic.xml [hunk 3]
```diff
@@ -30,8 +30,8 @@
 		<description>Light State</description>
 		<state readOnly="true">
 			<options>
-				<option value="0">Off</option>
-				<option value="1">On</option>
+				<option value="0">OFF</option>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/colorlogic.xml:33]: As label, "On" and "Off" are better than all in capitals. After all, option 7 is also called "Powering Off". So please revert this change.

**Review by kaikreuzer — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/backyard.xml (hunk 1 above)
Comment (fwolter) [inline bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/backyard.xml:36]:
```suggestion
		<state pattern="%.1f%unit%" readOnly="true"/>
```

**Review by fwolter — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/backyard.xml (hunk 1 above)
Comment (matchews) [reply bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/backyard.xml:36]: The Omni Logix server only reports integer values for temperature.  Do you still prefer %.1f %unit% ?

Comment (matchews) [reply bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/backyard.xml]: Also, any hints on how to handle the failed build on the org.openhab.persistence.dynamodb.internal.TestStoreMixedTypesTest ?

Comment (fwolter) [reply bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/backyard.xml]: I thought it was a typo. Integer would be `%d %unit`. The failed test seems unreleated to your changes.

**Review by fwolter — APPROVED** [review]
LGTM, seems Kai's comments are also addressed.
